addons/base_import/static/src/js/import_action.js

```diff
@@ -609,6 +623,50 @@ var DataImport = AbstractAction.extend({
         });
     },
 
+    /**
+     * Called when the user manually interrupts the import during a batched import.
+     * Sets 'stopImport' to true, which will stop the process when the current batch is done.
+     *
+     * @private
+     * @param {MouseEvent} event
+     */
+    _onStopImport: function (event) {
```

Comment:
Linting: onStop should be defined after onBatch (and probably after other methods, notably _clean...) .